test/core/fixtures/transformation/validation.undeclared-variable-check/undeclared-shorthand-property/exec.js

```diff
@@ -0,0 +1 @@
+({foo})
```

Comment:
Yes, `node === parent.value`, but also `node === parent.key`, which means that the `"Property"` case of `isReferenced` will only return true if it is a computed property, which it isn't.